cql3/statements/modification_statement.cc

```diff
@@ -439,13 +442,31 @@ modification_statement::process_where_clause(data_dictionary::database db, expr:
     }
 }
 
+::shared_ptr<strongly_consistent_modification_statement>
+modification_statement::prepare_for_raft() const {
+    // FIXME: implement for every type of `modification_statement`.
+    throw raft::broadcast_tables::unsupported_operation_error{};
+}
+
 namespace raw {
 
+::shared_ptr<cql_statement_opt_metadata>
+modification_statement::prepare_statement(data_dictionary::database db, prepare_context& ctx, cql_stats& stats) {
+    ::shared_ptr<cql3::statements::modification_statement> statement = prepare(db, ctx, stats);
+
+    if (db.get_config().check_experimental(db::experimental_features_t::feature::BROADCAST_TABLES) &&
+            raft::broadcast_tables::is_broadcast_table_statement(keyspace(), column_family())) [[unlikely]] {
```

Comment:
Isn't the experimental check unneeded? is_broadcast_table_statement() will return false if it's not enabled.

Minor: [[unlikely]] is unneeded, since prepare is rare.

Reply:
`is_broadcast_table_statement()` is meant to check only if the query should be executed on a broadcast table. I can change it to check also if this feature is enabled.

Reply:
I thought the table doesn't exist if the feature isn't enabled.

Reply:
Yes, you're right. I will fix it.

Reply:
Removed experimental check and [[unlikely]].